src/tests/helpers.ts

```diff
@@ -161,8 +161,8 @@ export function protocolLink(providerProtocol: ProviderProtocol = ProviderProtoc
   // eslint-disable-next-line no-undef
   const secret = process.env.UNSTOPPABLE_RESOLUTION_INFURA_PROJECTID;
   const protocolMap = {
-    [ProviderProtocol.http]: secret ? `https://mainnet.infura.io/v3/${secret}` : 'https://main-rpc.linkpool.io',
-    [ProviderProtocol.wss]: secret ? `wss://mainnet.infura.io/ws/v3/${secret}` : 'wss://main-rpc.linkpool.io/ws',
+    [ProviderProtocol.http]: secret ? `https://mainnet.infura.io/v3/${secret}` : 'https://mainnet.infura.io/v3/d423cf2499584d7fbe171e33b42cfbee',
```

Comment:
@JohnnyJumper Do we need to include infura migration to the resolution 2.0 release? It will be harder to test.